src/packages/@ncigdc/components/Modals/ControlledAccess/ControlledAccessModal.js

```diff
@@ -82,7 +82,7 @@ const ControlledAccessModal = ({
           handleProgramSelect,
           selectedStudies,
           setSelectedStudies,
-          studiesList,
+          studiesSummary,
```

Comment:
This one got renamed and the change was applied to the feature branch, but not onto mainstream.
Modal crashes if the flag is on.